.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  plugins: ['prettier'],
+  parser: '@typescript-eslint/parser',
+  extends: [
+    'plugin:react/recommended',
+    'plugin:@typescript-eslint/recommended',
+    'prettier/@typescript-eslint',
```

Comment:
This doesn't need the `plugin:`?

Reply:
yeah, probably not.  Will remove.  TY.

Reply:
To follow-up: `eslint-plugin-` prefix could be omitted, but the `plugin:` prefix [is needed](https://eslint.org/docs/user-guide/configuring#configuring-plugins)